preferences/src/main/kotlin/dev/sebastiano/bundel/preferences/PreferencesScreen.kt

```diff
@@ -1,4 +1,4 @@
-@file:OptIn(ExperimentalAnimationApi::class)
+@file:OptIn(ExperimentalAnimationApi::class, ExperimentalAnimationApi::class)
```

Comment:
Do we need the dupe?

Reply:
done. Stupid IDE recommended adding it and the red squiggly went - didn't realise it was dupe!

Reply:
ahahah